src/dcos_e2e/node.py

```diff
@@ -30,6 +36,8 @@ def __init__(self, ip_address: IPv4Address, ssh_key_path: Path) -> None:
         """
         self.ip_address = ip_address
         self._ssh_key_path = ssh_key_path
+        self.private_ip_address = private_ip_address if private_ip_address \
```

Comment:
Instead, I think it is better that `private_ip_address` is required. Happy to discuss.